File: Level_1/제일 작은 수 제거하기/Remove_min.java
class Solution {
    public int[] solution(int[] arr) {
        int[] answer = new int[arr.length - 1];
        int min = arr[0], index = 0;
        
        if(arr.length == 1){
           return new int[]{-1};
        }
        for(int i = 0; i < arr.length; i++){
            if(min > arr[i])
                min = arr[i];
        }
        for(int i = 0; i < arr.length; i++){
            if(min == arr[i]) continue;
            else answer[index++] = arr[i];
        }        
        return answer;
    }
}

File: Level_1/제일 작은 수 제거하기/Remove_min.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> arr) {
    vector<int> answer = arr;
    
    if(answer.size() == 1) return {-1};
    
    answer.erase(min_element(answer.begin(), answer.end()));

    return answer;
}
